### PR TITLE
Add project_name to CTS

### LINKED ISSUE #530: [Feature Request] CTS Tracker per project
Hi there,
please provide the possibility to set the project for a CTS tracker.

### Terraform Version
Terraform v0.12.26

### Affected Resource(s)
- resource_opentelekomcloud_cts_tracker_v1.go 
- and everything affected https://github.com/terraform-providers/terraform-provider-opentelekomcloud/pull/135/files

### Expected Behavior
I expect to be able to provide a project name to the tracker (one tracker named system can be set per project)

### Actual Behavior
Only adds a tracker to the base project. E.g. `eu-de`

### References
- GH-135

### PR DESCRIPTION
**Pull request by lego963**
Closes: #530

### CODE REVIEW

File: opentelekomcloud/util.go [hunk 1]
```diff
@@ -49,8 +49,17 @@ func GetRegion(d *schema.ResourceData, config *Config) string {
 	return strings.Split(n, "_")[0]
 }
 
+// GetProjectName returns the project name that was specified in the resource.
+func GetProjectName(d *schema.ResourceData, config *Config) string {
```
Comment (outcatcher) [inline opentelekomcloud/util.go:53]: Project name from the schema is never used

File: opentelekomcloud/resource_opentelekomcloud_cts_tracker_v1.go [hunk 2]
```diff
@@ -148,14 +153,15 @@ func resourceCTSTrackerRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("need_notify_user_list", ctsTracker.SimpleMessageNotification.NeedNotifyUserList)
 
 	d.Set("region", GetRegion(d, config))
+	d.Set("project_name", GetProjectName(d, config))
```
Comment (outcatcher) [inline opentelekomcloud/resource_opentelekomcloud_cts_tracker_v1.go]: Project name in schema is always overwritten by current project

**Review by outcatcher — CHANGES_REQUESTED** [review]

**Comment by lego963** [comment]
Need to add same possibility to smn_topic_v2

**Comment by lego963** [comment]
Close due to remove base repository

**Pull request closed without merging**